package.json

```diff
@@ -31,9 +31,10 @@
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
     "docs": "jsdoc ./src/nwbuild.js -d docs",
     "demo:cli": "cd ./test/demo && nwbuild ./nwapp --mode=build --version=0.70.1 --flavour=normal --platform=win --arch=x64 --outDir=./build",
-    "demo:bld": "cd ./test/demo && node bld.cjs",
-    "demo:run": "cd ./test/demo && node run.cjs",
-    "cjs": "esbuild ./src/nwbuild.js --bundle --minify --legal-comments=inline --platform=node --outfile=./nwbuild.cjs",
+    "demo:nux": "npm run cjs && cd ./test/demo && node nux.js",
```

Comment:
IMO "nux" is a terrible name for Linux/UNIX (often abbreviated as "*nix" but never "*nux"). I recommend sticking with "linux".